swc-uxp-starter/README.md

```diff
@@ -0,0 +1,161 @@
+
+**Requisite imports**
+
+```javascript
+//sp-theme
+
+import '@spectrum-web-components/theme/sp-theme.js';
+
+
+//themes to use classic and spectrum themes
+
+import '@spectrum-web-components/theme/src/themes.js';
+
+import '@spectrum-web-components/theme/src/express/themes.js';
+```
+
+<img width="1239" alt="image2023-1-20_19-29-43" src="https://git.corp.adobe.com/storage/user/38751/files/6f7b68ec-8eeb-4776-85c9-bcc9bc43002d">
+
+
+
+# **4. Frequently Asked Questions**
+1. Do I need to install specified versions of the components? 
+   👉🏼 Yes. It is recommended to use the specified versions. There could be inter dependancy of the components and one component could get impacted if the other upgrades.  The [**wiki**](https://wiki.corp.adobe.com/display/UXP/Support+for+Spectrum+Web+Components+in+UXP) would keep getting updated for the supported versions of the components.
+2. Do I need to wrap all the components in <sp-theme> ?
+   👉🏼 Yes. <sp-theme> is the one which delivers the spectrum style tokens to the wrapped html.
+   💡 Also make sure to import the theme data parts as mentioned in point 6.
+3. Why do I need the resolution block in the package.json?
+   👉🏼 In order to resolve transitive dependancies we need to [force resolutions](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/). Thereby to avoid incompatibility issues in sub components, one needs to mention all the dependancies in the resolution block too.
+4. I am facing error like the following in the console. How do I resolve this ?
+
+```
+Failed to execute 'define' on 'CustomElementRegistry': the name "sp-icon-alert" has already been used with this registry
+```
+
+* 👉🏼 Errors like this occur when the package manager has resolved more than one copy of component (sp-icon-alert in this case) package. You can check on the following when face this issue.
+     * a. Make sure you dont have multiple calls to bundle.js somehow.
+     * b. Check if you have added the component in the resolutions block in package.json.
+     * c. Check if you are using correct recommended version of the package without the ^.
+5. Where do I find sample plugins for the reference?
+   👉🏼 You can refer to different branches of this [repo](https://git.corp.adobe.com/panbhati/swc-uxp-starter-plugin).
+
+
+# **5. Supported Components**
+Following are the components with their variants supported along with the versions. Text color of the variants represent the state as in : </br>
+🟢 👉🏼 Variant is working as per the specs of latest  SWC(web) version. </br>
+🟠 👉🏼 Variant is working as per latest specs however minor styling issues present with a workaround that can be applied in UXP plugin.</br>
+🔴 👉🏼 Variant has issues due to missing UXP core platform features. Will be addressed only in later releases.</br>
+
+| Components      | Variants with status | Version |
+| --------------- | -------------------- | ------- |
+| Action-Bar      |   Fixed 🟢    </br>   Sticky 🔴      |      0.5.9 |
```

Comment:
Do we have the link to this specific version so that the developers know what is expected behavior?